app/authentication/authenticator.py

```diff
@@ -81,14 +82,14 @@ def decrypt_token(encrypted_token):
     if not encrypted_token or encrypted_token is None:
         raise NoTokenException("Please provide a token")
 
-    jwt_token = decrypt_jwe(encrypted_token=encrypted_token,
-                            secret_store=current_app.eq['secret_store'],
-                            purpose=KEY_PURPOSE_AUTHENTICATION)
+    jwt_token = JWEHelper.decrypt(encrypted_token=encrypted_token,
+                                  key_store=current_app.eq['secret_store'],
+                                  purpose=KEY_PURPOSE_AUTHENTICATION)
 
-    decrypted_token = decode_jwt(jwt_token=jwt_token,
-                                 secret_store=current_app.eq['secret_store'],
-                                 purpose=KEY_PURPOSE_AUTHENTICATION,
-                                 leeway=current_app.config['EQ_JWT_LEEWAY_IN_SECONDS'])
+    decrypted_token = JWTHelper.decode(jwt_token=jwt_token,
+                                       key_store=current_app.eq['secret_store'],
+                                       purpose=KEY_PURPOSE_AUTHENTICATION,
+                                       leeway=current_app.config['EQ_JWT_LEEWAY_IN_SECONDS'])
```

Comment:
This repeated pattern of `decrypt` -> `decode` can be dropped in favour of the helper function from the lib, `sdc.crypto.decrypter.decrypt`.